Documentation/ApiOverview/Backend/JavaScript/ES6/Index.rst

```diff
@@ -123,7 +123,7 @@ Migration from RequireJS
 
 ..  note:: For general information on how to migrate from AMD (RequireJS) to
     ES6 see this article: `Arthur Yidi: Migrate JavaScript Modules From AMD
-    to ES6 <https://arthuryidi.com/migrate-amd-modules/>`__. This is about
+    to ES6 <https://web.archive.org/web/20220516095256/https://arthuryidi.com/migrate-amd-modules/>`__. This is about
```

Comment:
Something is odd about the url now?

Reply:
The URL is valid like this. web.archive.org is a service mirroring sites. The URL points to the original site at a certain point in time when archive.org did a scan of that site.

Visit it: https://web.archive.org/web/20220516095256/https://arthuryidi.com/migrate-amd-modules/